### ISSUE TITLE
Rework log4pot extraction process

### ISSUE THREAD

**Issue by regulartim** [intelowlproject/GreedyBear#410]
In #407 the extraction process for cowrie and general honeypots was changed to be able to fill the new IOC model fields. This was not done for log4pot yet.

**Comment by mlodic** [comment]
The log4pot does not generate much information at this time, it was more useful once that vulnerability came out. We can consider this task as low priority

**Comment by regulartim** [comment]
Agree. It is also deactivated by default in current versions of TPot.
